packages/webapp/src/elections/api/eden-contract.ts

```diff
@@ -0,0 +1,32 @@
+import { devUseFixtureData } from "config";
+import { ElectionState } from "elections/interfaces";
+import {
+    CONTRACT_CURRENT_ELECTION_TABLE,
+    CONTRACT_ELECTION_STATE_TABLE,
+    getTableRows,
+} from "_app";
+import { fixtureElectionState } from "./fixtures";
+
+export const getCurrentElection = async () => {
+    const rows = await getTableRows<any>(CONTRACT_CURRENT_ELECTION_TABLE);
```

Comment:
Do you also want a `if (devUseFixtureData) return fixtureCurrentElection;` here?

Reply:
Yes, just didn't get to that yet; added a TODO to indicate that